tests/test_vqe.py

```diff
@@ -294,6 +294,18 @@ def test_cost_invalid_ansatz(self, ansatz, mock_device):
             cost = qml.vqe.cost([], 4, hamiltonian, mock_device)
 
 
+expected_temp_grad = np.array(
```

Comment:
How was this established?

Reply:
Yep, this is a bad practice on my part. I want to check that all three interfaces return the same gradient for the same test circuit, not so much that they return the _correct_ gradient. Is there a better way of checking this?

Reply:
I suppose I can move this new test for each interface into a single test.

Reply:
Use numpy interface as baseline, and make sure others agree

Reply:
Done in da53fd9

Reply:
18 seconds. That was speedy :)